scss/mixins/_buttons.scss

```diff
@@ -46,12 +47,11 @@
 
   &.disabled,
   &:disabled {
-    &:focus,
-    &.focus {
+    @include hover-focus {
       background-color: $background;
-          border-color: $border;
+          border-color: $border;    
```

Comment:
Line should be indented 6 spaces, but was indented 10 spaces<br>Line contains trailing whitespace